agent/connect/ca/plugin/provider.proto

```diff
@@ -79,6 +81,14 @@ message CrossSignCAResponse {
     string crt_pem = 1;
 }
 
+message SupportsCrossSigningResponse {
+    bool supports_cross_signing = 1;
+}
+
+message MinLifetimeResponse {
+    string min_lifetime = 1;
```

Comment:
Why a string? Protobuf support durations that auto decode into `time.Duration`. https://developers.google.com/protocol-buffers/docs/reference/csharp/class/google/protobuf/well-known-types/duration.html#class_google_1_1_protobuf_1_1_well_known_types_1_1_duration maybe there is a good reason to use a string though?

Reply:
This is my first time using protobuf, and I missed the fact that it supports `Duration` as a type. Will fix.

Reply:
Fixed in 7fb8d6e525334ce04c28777fa1412220afd95d9d.